### ISSUE TITLE
diarization develop branch

### ISSUE THREAD

**Issue by RickyGunawan09** [pyannote/pyannote-audio#837]
I just wanna ask about how to use it, I try your example for using Diarization pipeline but i get nothing. 
but when I try another pipeline like VAD, OSD or Resegmentation they run normally and i can get the segment.

I'm using anaconda like you recommend in example.

is there an error or am I using it wrong?
thank you

**Comment by hbredin** [comment]
Would you mind sharing a reproducible example ? 
* your audio file
* what worked
* what did not work

**Comment by RickyGunawan09** [comment]
thank you for answering.

wait let me share to you the code I'm testing and the audio file I use. 
in my case nothing it's working for diarization

[https://drive.google.com/drive/folders/1RH5gD-ABFxBP9Sjygki6-df-NPhkpkkT?usp=sharing](url)
in this google drive there is a code I use for testing diarization, overlap,resegmentation, and vad 
and there is two audio that I use for testing it (the audio file using indonesian language)

there is a screenshot for every code result and conda list from my environment pyannote.

the strange behavior it's only for diarization. and when I run diarization for two times in a row it stuck like in my screenshot.

thank you

**Comment by hbredin** [comment]
Reducing `min_activity` to something like 3 should do the trick (instead of using [default parameters](https://github.com/pyannote/pyannote-audio/blob/0a368c6bcb72ded4a080f72ed945e2dd26462271/pyannote/audio/pipelines/speaker_diarization.py#L163-L171)).

```python
pipeline = Pipeline.from_pretrained("pyannote/speaker-diarization")
pipeline.instantiate({
   "onset": 0.810,
   "offset": 0.481,
   "min_duration_on": 0.055,
   "min_duration_off": 0.098,
   "min_activity": 3.0,   # <==== here
   "stitch_threshold": 0.040,
   "clustering": {"method": "average", "threshold": 0.595}})
diarization = pipeline("voip.wav")
```

![image](https://user-images.githubusercontent.com/1080837/149223173-9194c99d-0f2f-45a4-9f3e-9a37c763ec83.png)

**Comment by hbredin** [comment]
Fixed in 767b350f43dadf4c65f0c5d203aeaef3bb6362a9
